storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -0,0 +1,32 @@
+[id="persistent-storage-vsphere"]
+= vSphere CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-vsphere
+
+toc::[]
+
+== Overview
+
+{product-title} can provision persistent volumes (PVs) using the Container Storage Interface (CSI) VMware vSphere driver for Virtual Machine Disk (VMDK) volumes.
+
+:FeatureName: vSphere CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned persistent volumes (PVs) that mount to vSphere storage assets, {product-title} installs the vSphere CSI Driver Operator and the vSphere CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+
+* *vSphere CSI Driver Operator*: By default, the Operator provides a storage class that you can use to create persistent volumes claims (PVCs). You also have the option to create the vSphere storage class as described in xref:../../storage/persistent_storage/persistent-storage-vsphere.adoc#persistent-storage-using-vsphere[Persistent storage using vSphere].
+
```

Comment:
I don't think we can link to intree storage driver docs. The CSI driver is very different from, Can we link upstream docs - https://vsphere-csi-driver.sigs.k8s.io/features/block_volume.html ?

Reply:
I'm reluctant to link to an outside source for content. I've decided to recreate the relevant content within our repo. Hope that works for you.